content/departments/product-engineering/engineering/cloud/delivery/index.md

```diff
@@ -16,32 +16,56 @@ Enable any Sourcegraph customer or user to trial or run (in production) Sourcegr
 
 ## Responsibilities
 
-- Deployments methods
+- Deployments methods (e.g. docker, kustomize, helm)
 - [Customer deployments and docs](https://docs.sourcegraph.com/admin/install)
 - [Managed instances](./managed/index.md)
-- ~~[Sourcegraph releases](../../process/releases/index.md)~~
+- ~~[Sourcegraph releases](../../process/releases/index.md)~~ releases are owned by the [release guild](../../process/releases/release_guild.md)
```

Comment:
This may be a little more nuanced. The release guild is not a formal organization. I believe the Delivery team is still responsible for releases and, in the current structure, Delivery provides support to the release guild.